internal/pubsub/example_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pubsub_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/google/go-cloud/internal/pubsub"
+	"github.com/google/go-cloud/internal/pubsub/mempubsub"
+)
+
+func ExampleSendReceive() {
```

Comment:
IIUC, this should just be called `Example`. Or `Example_sendReceive`.

It would be good to add an example with a more realistic `Receive` loop too, can be separate PR if you prefer.

Reply:
Changed the test name.

Filed #738.